app/views/v1/sessions/destroy.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+json.session do
+  if @user
+    json.success true
+    json.email @user.email
+    json.auth_token @user.auth_token
+  else
```

Comment:
I think this case is not necessary for `destroy` action. 
1) `SessionsController#destroy` has a call to `ApplicationController#authenticate` through a `before_action`. 
2) `authenticate` calls to `SessionsController#authenticate_token` checking the existence of email and token. 
So:
`User.find_by(email: email, auth_token: token)` from `authenticate_token` will actually check if the user exists, so we won't reach else case here.